### ISSUE TITLE
Which node version supported?

### ISSUE THREAD

**Issue by Semigradsky** [hypermodules/gh-release-assets#14]
From changelog:
```
2.0.0

- BREAKING CHANGE: Update all deps which raises the minimum node version.
```

but there is no [`engines` field in `package.json`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines).

**Comment by bcomnes** [comment]
Sorry, didn't have time to figure it out exactly. LTS is assumed and we should specify it in package.json. If it's not working on LTS it should be an issue (I think it is).

**Comment by ungoldman** [comment]
Hey @Semigradsky, sorry for the confusion. The minimum node version (12) is now explicitly set in the `engines` field in `package.json` (https://github.com/ungoldman/gh-release-assets/pull/17).

We are generally following the LTS maintenance schedule outlined in the official Node.js docs here:

https://nodejs.org/en/about/releases/

**Comment by ungoldman** [comment]
Fix released on npm in [`v2.0.1`](https://github.com/ungoldman/gh-release-assets/releases/tag/v2.0.1)

**Comment by Semigradsky** [comment]
Looks like `util-extend` dependency can be replaced by native `Object.assign`.
